sn_transfers/src/genesis.rs

```diff
@@ -66,10 +67,19 @@ lazy_static! {
     /// The hard coded value is for production release, allows all nodes to validate it.
```

Comment:
comment needs to be updated once this PR merged.
actually, I think this block of comment can be `removed` totally.
it was a leftover from the previous refactor.
the topic is already covered by the comment of the correspondent str const